inst/@sym/mpower.m

```diff
@@ -129,7 +129,7 @@
 %! C = 10 + B + B^2;
 %! D = subs(C, n, 1);
 %! E = 10 + A + A^2;
-%! assert (isequal (D, E))
+%! assert (isequal (simplify(D), simplify(E)))
```

Comment:
its need, some versions of sympy give different results and fails, simplify fix it.